docs/usage_statistics_exports.md

```diff
@@ -0,0 +1,25 @@
+# How can you export data
+
+Aggregated anonymous daily data can be extracted from the application for any
+specified time period using the `/data-export` route.  Output is available in
+CSV format.
+
+For example, to get data for a five day period starting 10th April 2020:
+
+     <path-to-service>/data-export.csv?start_date=2020-04-08&end_date=2020-04-15
```

Comment:
I know you've just copied this from the README, but the date range gives seven days rather than five days (as mentioned above)

Reply:
Great spot!